contracts/tokens/ArrowVestingBase.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Valid import paths for hardhat (need to comment out when using brownie)
+// import "@openzeppelin/contracts-upgradeable/finance/VestingWalletUpgradeable.sol";
+// import "@openzeppelin/contracts-upgradeable/access/AccessControlEnumerableUpgradeable.sol";
+// import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+// import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+// import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+// For brownie tests (should remove once we fully move to hardhat)
+import "@openzeppelin-upgradeable/contracts/finance/VestingWalletUpgradeable.sol";
+import "@openzeppelin-upgradeable/contracts/access/AccessControlEnumerableUpgradeable.sol";
+import "@openzeppelin-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import "@openzeppelin-upgradeable/contracts/proxy/utils/Initializable.sol";
+import "@openzeppelin-upgradeable/contracts/proxy/utils/UUPSUpgradeable.sol";
+
+/// @title Base implementation of vesting contract that will be cloned into individual vesting wallets pointing at specific beneficiary addresses.
+contract ArrowVestingBase is
```

Comment:
I don't see why this contract needs to be upgradable. The OpenZeppelin contract already has all the functionality we need, and we can always deploy a new `ArrowVestingFactory` contract in the very worst case.

Reply:
I thought this contract is already upgradeable? https://github.com/Arrow-air/Arrow-Contracts/blob/main/contracts/ArrowVestingBase.sol

All I changed was to make it conform to UUPS proxies.